Modules/permissions.py

```diff
@@ -128,4 +127,67 @@ def real_decorator(func):
                                     else permission.denied_message)
 
         return guarded
+
+    return real_decorator
+
+
+def require_channel(message: str = "This command must be invoked in a channel."):
```

Comment:
It might prove a friendly piece of user interface for the failure message to include the invoked command's name in case multiple commands are invoked in quick succession?
I would also drop the parameter completely and make the decorator direct, but that is debatable.

Reply:
as with the permission check decorator, i am leaving the option open for custom error messages. as for the command name? that should be pullable from the Context object. il look into this

Reply:
Done.